src/recipe/Baked.Recipe.Service.Application/Ui/UiLayer.cs

```diff
@@ -9,19 +9,37 @@ namespace Baked.Ui;
 
 public class UiLayer : LayerBase<GenerateCode>
 {
+    public delegate string NewLocale(string key, string? value = default);
```

Comment:
this doesn't create a new locale. it adds a new key to a locale (template). 

maybe `NewLocaleKey` is better 🤔